py4syn/utils/counter.py

```diff
@@ -219,6 +219,9 @@ def enableCounter(mne):
     c = py4syn.counterDB[mne]
     c['enable'] = True
 
+def delCounters():
```

Comment:
Please rename it to something like:
```
def clearCountersDB()
```

Del can lead to a different understanding of it.